test/specs/pushAndPull.e2e.ts

```diff
@@ -88,21 +65,32 @@ describe('Push and Pull', async () => {
     expect(filteredTreeViewItems.includes('ExampleApexClass1.cls-meta.xml')).toBe(true);
   });
 
-  step('Push the Apex class', async () => {
+  step('SFDX: View Local Changes', async () => {
     const workbench = await (await browser.getWorkbench()).wait();
-    await utilities.runCommandFromCommandPrompt(workbench, 'SFDX: Push Source to Default Org', 5);
+    await utilities.runCommandFromCommandPrompt(workbench, 'SFDX: View Local Changes', 5);
 
-    // At this point there should be no conflicts since this is a new class.
+    // Check the output.
+    const outputPanelText = await utilities.attemptToFindOutputPanelText(
+      `Salesforce CLI`,
+      `Source Status`,
+      10
+    );
 
-    const successNotificationWasFound = await utilities.notificationIsPresentWithTimeout(
-      workbench,
-      'SFDX: Push Source to Default Org successfully ran',
-      utilities.TEN_MINUTES
+    expect(outputPanelText).toContain(
+      `Local Add  ExampleApexClass1  ApexClass  ${path.join('force-app', 'main', 'default', 'classes', 'ExampleApexClass1.cls')}`
     );
-    expect(successNotificationWasFound).toBe(true);
```

Comment:
Why don't we want to check for the presence of the success notification anymore?

Reply:
I will restore it and retain the additional checks in the output tab.
I thought the output tab would be a little more stable, but ignored that the notification is part of the app as well and needs to be checked.